OctopusDSC/Tests/OctopusServerExeInvocationFiles/ConfigureWhenMsiPreInstalled/CurrentState.ps1

```diff
@@ -0,0 +1,8 @@
+[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingConvertToSecureStringWithPlainText', '')] # these are tests, not anything that needs to be secure
+param()
+
+return @{
+  Ensure = "Present";
+  State = "Installed";
+  DownloadUrl = "https://octopus-testing.s3.amazonaws.com/server/Octopus.2018.2.7-x64.msi"
```

Comment:
Could we point this to `https://octopus.com/downloads/latest/WindowsX64/OctopusServer` so that it always downloads the latest MSI?

Reply:
It doesn't make any difference - it's never actually downloaded (the actual download is mocked)